cmd/finch/version_test.go

```diff
@@ -5,44 +5,177 @@ package main
 
 import (
 	"bytes"
-	"io"
-	"os"
+	"errors"
+	"strings"
 	"testing"
 
+	"github.com/spf13/cobra"
+
+	"github.com/runfinch/finch/pkg/mocks"
+
+	"github.com/golang/mock/gomock"
 	"github.com/stretchr/testify/assert"
-	"github.com/stretchr/testify/require"
 )
 
-func captureStdout(t *testing.T, f func()) string {
-	temp := os.Stdout
-	read, write, _ := os.Pipe()
-	os.Stdout = write
+func TestVersionCommand(t *testing.T) {
```

Comment:
To make the name of the testing function align with the name of the tested function.

```suggestion
func TestNewVersionCommand(t *testing.T) {
```